packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -266,6 +282,23 @@ async function createWithdrawalEvent(
   })
 }
 
+async function deactivateOpenIncomingPaymentsByPaymentPointer(
+  deps: ServiceDependencies,
+  paymentPointerId: string
+) {
+  await IncomingPayment.query(deps.knex)
+    .patch({
+      expiresAt: new Date(
+        Date.now() + deps.config.paymentPointerDeactivationPaymentGracePeriodMs
+      )
+    })
+    .where('paymentPointerId', paymentPointerId)
+    .whereIn('state', [
+      IncomingPaymentState.Pending,
+      IncomingPaymentState.Processing
+    ])
```

Comment:
I think after the 90 day default changes, we _should_ include expiryDate as a `where` clause in here

Reply:
Still need this I believe ^

Reply:
@mkurapov added